cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -1136,3 +1136,33 @@ def test_dialog_default_value_selection(appliance, custom_categories, import_dat
         view.fields("network").read() == network and
         view.fields("number_disk").read() == additional_disks
     )
+
+
+@pytest.mark.meta(automates=[1783375])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize("file_name", ["bz_1783375.yml"], ids=["sample_dialog"],)
+def test_dialog_not_required_default_value(appliance, generic_catalog_item_with_imported_dialog,
+                                           file_name):
+    """
+    Bugzilla:
+        1783375
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/16h
+        startsin: 5.10
+    """
+    catalog_item, _, ele_label = generic_catalog_item_with_imported_dialog
+    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+
+    # download yaml file
+    fs = FTPClientWrapper(cfme_data.ftpserver.entities.dialogs)
```

Comment:
Optional:
We can merge line 1159 and 1160 as we are not using more than one time as below:
`file_path = FTPClientWrapper(cfme_data.ftpserver.entities.dialogs).download(file_name)`